### PR TITLE
Small fixes (typos, comments) found running through the tutorial

### PR DESCRIPTION
**Pull request by t-bast**
Great job on the tutorial!
Here are the small typos/markdown stuff I ran against while following it (my markdown linter did most of the work).

I took some notes on what went through my mind / potential improvements, take them with a grain of salt but I hope it can be useful to make it even better:

**General**

The `Edit on Github` links don't seem to work (probably because you expect to move the tutorial inside the cosmos-sdk repo).

**Start your application**

"Commit if the ABCI message is DeliverTx" -> isn't this a bit misleading? DeliverTx should only be adding to the tx pool right? If DeliverTx is rejected or found invalid at block commit time bad things might happen.

Maybe mention that the user should do a `go get ./...` to fetch dependencies (or `go get github.com/tendermint/tendermint` and `go get github.com/cosmos/cosmos-sdk`). Later you introduce `dep`, but maybe it's worth introducing it at the start?

Codec not defined at that point so project not building, maybe it makes sense to start with the x/nameservice module and then create the app.go (because `MakeCodec()` isn't defined).

**BuyName**

`Bid.IsPositive()` doesn't exist anymore. I replaced by `Bid.IsAllPositive()`

**Codec File**

Maybe explain why the `WhoIs` struct doesn't need to be registered (because it's not included in any transaction and doesn't need canonical marshalling, it's just used in CLI RPC).

**Building and running the application**

This part has some redundant parts with the previous ones (about `dep`, building and installing).
I think they can be removed.

**Conclusion**

Maybe a small conclusion paragraph that lists some potentially interesting apps that can be built?
Or a list of known projects building on top of the Cosmos SDK?

### CODE REVIEW

File: tutorial/app-init.md [hunk 1]
```diff
@@ -4,7 +4,12 @@ Get started by creating a new file: `./app.go`. This file is the heart of your d
 
 In `app.go`, you define what the application does when it receives a transaction. But first, it needs to be able to receive transactions in the correct order. This is the role of the [Tendermint consensus engine](https://github.com/tendermint/tendermint).
 
-Start by importing the necessary dependencies:
+Start by installing and importing the necessary dependencies:
+
+```sh
```
Comment (yangyanqing) [inline tutorial/app-init.md]: I don't think the following two steps were necessary, because they would be call while running `make get_vendor_deps`.

Comment (t-bast) [reply tutorial/app-init.md]: Once we reach the end of the tutorial (where we start using deps) I agree :)
But it was a bit weird to follow the tutorial with something that couldn't build, which is why I added that...
Maybe the dep installation should be in the first steps of the tutorial (but that should probably be done in another PR)?

Comment (jackzampolin) [reply tutorial/app-init.md]: I don't think we need this here.

Comment (t-bast) [reply tutorial/app-init.md]: Allright I'll remove it then, maybe another change to the tutorial can bring `dep` closer to the introduction.

**Comment by t-bast** [comment]
Sure I'll re-configure my editor and remove the whitespace changes, it makes more sense to be in a dedicated PR of its own.

File: tutorial/cli.md [hunk 2]
```diff
@@ -211,37 +212,38 @@ func NewModuleClient(storeKey string, cdc *amino.Codec) ModuleClient {
 
 // GetQueryCmd returns the cli query commands for this module
 func (mc ModuleClient) GetQueryCmd() *cobra.Command {
-	// Group gov queries under a subcommand
-	govQueryCmd := &cobra.Command{
+	// Group nameservice queries under a subcommand
```
Comment (jackzampolin) [inline tutorial/cli.md]: Thank you for catching my copy pasta here 🙈

Comment (jackzampolin) [reply tutorial/cli.md]: If you are making changes in the code in the `.md` files, please also make the same change in the corresponding `.go` file!

File: tutorial/buy-name.md [hunk 3]
```diff
@@ -21,27 +21,27 @@ func NewMsgBuyName(name string, bid sdk.Coins, buyer sdk.AccAddress) MsgBuyName
 	}
 }
 
-// Type Implements Msg.
+// Route should return the name of the module
```
Comment (jackzampolin) [inline tutorial/buy-name.md]: If you are making a change in the code in the `.md` files, please make a corresponding change in the `.go` file.

File: tutorial/entrypoint.md [hunk 4]
```diff
@@ -250,10 +250,8 @@ func SimpleAppGenTx(cdc *codec.Codec, pk crypto.PubKey) (
 ```
 
 Notes on the above code:
-- Most of the code above combines the CLI commands from
-	1. Tendermint
-	2. Cosmos-SDK
-	3. Your Nameservice module
+
+- Most of the code above combines the CLI commands from 1. Tendermint 2. Cosmos-SDK 3. Your Nameservice module
```
Comment (jackzampolin) [inline tutorial/entrypoint.md]: You can remove the numbers here.

File: tutorial/entrypoint.md [hunk 5]
```diff
@@ -425,10 +423,8 @@ func initConfig(cmd *cobra.Command) error {
 ```
 
 Note:
-- The code combines the CLI commands from:
-	1. Tendermint
-	2. Cosmos-SDK
-	3. Your Nameservice module
+
+- The code combines the CLI commands from: 1. Tendermint 2. Cosmos-SDK 3. Your Nameservice module
```
Comment (jackzampolin) [inline tutorial/entrypoint.md]: ditto

File: tutorial/queriers.md [hunk 6]
```diff
@@ -17,7 +17,7 @@ import (
 	abci "github.com/tendermint/tendermint/abci/types"
 )
 
-// query endpoints supported by the governance Querier
+// query endpoints supported by the nameservice Querier
```
Comment (jackzampolin) [inline tutorial/queriers.md]: If you are making a change in the code in the `.md` files, please make a corresponding change in the `.go` file.

File: tutorial/set-name.md [hunk 7]
```diff
@@ -60,10 +61,10 @@ func (msg MsgSetName) ValidateBasic() sdk.Error {
 }
 ```
 
-`ValidateBasic` is used to provide some basic **stateless** checks on the validity of the `Msg`.  In this case, check that none of the attributes are empty. Note the use of the `sdk.Error` types here. The SDK provides a set of error types that are frequently encountered by application developers.
+`ValidateBasic` is used to provide some basic **stateless** checks on the validity of the `Msg`. In this case, check that none of the attributes are empty. Note the use of the `sdk.Error` types here. The SDK provides a set of error types that are frequently encountered by application developers.
 
 ```go
-// GetSignBytes Implements Msg.
+// GetSignBytes encodes the message for signing
```
Comment (jackzampolin) [inline tutorial/set-name.md]: If you are making a change in the code in the `.md` files, please make a corresponding change in the `.go` file.

File: tutorial/set-name.md [hunk 8]
```diff
@@ -115,12 +116,12 @@ func NewHandler(keeper Keeper) sdk.Handler {
...
 `NewHandler` is essentially a sub-router that directs messages coming into this module to the proper handler. At the moment, there is only one `Msg`/`Handler`.
 
-Now, you need to define the actual logic for handling the `MsgSetName` message in `handleMsgSetName`.:
+Now, you need to define the actual logic for handling the `MsgSetName` message in `handleMsgSetName`:
 
 > _*NOTE*_: The naming convention for handler names in the SDK is `handleMsg{ .Action }`
 
 ```go
-// Handle MsgSetName
+// Handle a message to set name
```
Comment (jackzampolin) [inline tutorial/set-name.md]: If you are making a change in the code in the `.md` files, please make a corresponding change in the `.go` file.

File: tutorial/buy-name.md (hunk 3 above)
Comment (t-bast) [reply tutorial/buy-name.md]: That does sound logical ;)

**Comment by jackzampolin** [comment]
Thank you for fixing that @t-bast!

**Pull request merged**